test/docTest.js.html

```diff
@@ -510,6 +514,22 @@
               $(".insert-filter").insertAfter(reference);
             }, /without parent/);
           });
+
+          it("should not throw error when off'ing an event that was not registered yet", function() {
```

Comment:
Does this test fail when the function off throw an error? If not, is it really needed?